src/styles/index.css

```diff
@@ -0,0 +1,145 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+}
+.header__logo {
+  font-size: 1.5rem;
+  font-weight: bold;
+  color: #060b35;
+  width: 40px;
+  height: 40px;
+}
+.header__nav-list {
+  display: flex;
+  gap: 20px;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+.header__nav-link {
+  color: #060b35;
+  text-decoration: none;
+  transition: color 300ms;
+}
+.header__nav-link:hover {
+  color: #00acdc;
+}
+.header__nav .is-active {
+  border-bottom: 2px solid #00acdc;
+}
+
+.card {
+  width: 200px;
+  padding: 20px;
+  border: 1px solid #ddd;
+  border-radius: 8px;
+  font-family: Arial, sans-serif;
+  text-align: center;
+}
+.card__image {
+  max-width: 100%;
+  border-radius: 4px;
+}
+.card__title {
+  font-weight: 700;
+  font-size: 1.1em;
+  margin: 10px 0;
+}
+.card__code, .card__reviews, .card__price {
+  font-weight: 400;
+  font-size: 0.9em;
+  color: #555;
+}
+.card__price-value {
+  font-weight: 700;
+  font-size: 1.2em;
+  color: #007bff;
+}
+.card__buy {
+  display: block;
+  width: 100%;
+  padding: 10px 0;
+  color: #007bff;
+  border: 2px solid #007bff;
+  border-radius: 4px;
+  font-weight: 700;
+  text-align: center;
+  text-decoration: none;
+  margin-top: 10px;
+  transition: background-color 0.3s, color 0.3s;
+}
+.card__buy:hover {
+  background-color: #007bff;
+  color: #fff;
+}
+
+.stars {
+  display: flex;
+}
+.stars__star {
+  width: 20px;
+  height: 20px;
+  background-image: url("../images/star.svg");
+  background-size: cover;
+}
+.stars__star.active {
+  background-image: url("../images/star-active.svg");
+}
+
+.catalog {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 48px;
+  padding: 50px 40px;
+}
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+}
+.header__logo {
+  font-size: 1.5rem;
+  font-weight: bold;
+  color: #060b35;
+  width: 40px;
+  height: 40px;
+}
+.header__nav-list {
+  display: flex;
+  gap: 20px;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+.header__nav-link {
+  color: #060b35;
+  text-decoration: none;
+  transition: color 300ms;
+}
+.header__nav-link:hover {
+  color: #00acdc;
+}
+.header__nav .is-active {
+  border-bottom: 2px solid #00acdc;
+}
```

Comment:
The styles for the header component are duplicated. Consider removing the redundant styles to keep the CSS file clean and efficient.